api/lib/infrastructure/data/user.js

```diff
@@ -1,6 +1,7 @@
-const Bookshelf = require('../bookshelf');
 const User = require('../../domain/models/User');
 const PixRole = require('../../domain/models/PixRole');
+
+const Bookshelf = require('../bookshelf');
```

Comment:
Remarque
Si seulement .. 
```suggestion
const Bookshelf = require('../bookshelf-validator');
```
ou
```suggestion
const Bookshelf = require('../validator/bookshelf');
```

Je verrais qu'on ne parle pas d'un package npm

Reply:
Je n'ai pas compris. Quel est le rapport avec validator ?